packages/neon_framework/packages/account_repository/test/account_storage_test.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:account_repository/account_repository.dart';
+import 'package:built_collection/built_collection.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:neon_framework/storage.dart';
+import 'package:test/test.dart';
+
+// ignore: avoid_implementing_value_types
+class _FakeBuiltList extends Fake implements BuiltList<String> {}
+
+class _SingleValueStoreMock extends Mock implements SingleValueStore {}
+
+void main() {
+  late SingleValueStore accounts;
+  late SingleValueStore lastAccount;
+  late AccountStorage storage;
+
+  setUp(() {
+    registerFallbackValue(_FakeBuiltList());
+
+    accounts = _SingleValueStoreMock();
+    lastAccount = _SingleValueStoreMock();
```

Comment:
```suggestion
    accountsStore = _SingleValueStoreMock();
    lastAccountStore = _SingleValueStoreMock();
```
These names are really confusing when reading the tests